test/units/playbook/test_collectionsearch.py

```diff
@@ -0,0 +1,42 @@
+# (c) 2020 Ansible Project
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from units.compat import unittest
+from ansible.playbook.collectionsearch import CollectionSearch
+from units.compat.mock import patch
+
+
+class TestCollectionSearch(unittest.TestCase):
```

Comment:
Inheriting from `unittest.TestCase` is harmful b/c our testrunner is `pytest` and this conflicts with a part of its features.